### ISSUE TITLE
Questions. Newbie, tho...

### ISSUE THREAD

**Issue by Paulie420** [nik012003/ripdrag#17]
I so wanna get involved in OSS - is main.rs the entire ripdrag program? Its useful, but I normally go the OTHER way - I want to drag INTO the folder... 

Thanks for creating, what other features do YOU want to add/work on?

**Comment by Paulie420** [comment]
I'd like to be able to right-click some file in ripdrag and have other options pop-up...

**Comment by nik012003** [comment]
Yes, main.rs is the entire program! If you wanna drag stuff inside the folder, target mode should work in combination with a shell script, but if you want you could try to implement other options!

Other stuff that's to be done are:
- Packaging for distros ( so .deb, .rpm , arch linux aur, ecc...)
- Better thumbnail implementation ( maybe look at how other gtk file managers get their thumbnails, since my implementation slows down when there are big photos/video files)

So feel free to do any of things I've listed, or your own ideas!
